cms/templates/cms/home_page.html

```diff
@@ -44,97 +45,134 @@
                 {% if program.programpage %}
-                	{% if program.programpage.external_program_page_url %}
-                	<a href="{{ program.programpage.external_program_page_url }}" class="program-link">
-                	{% else %}
-                  	<a href="{{ program.programpage.url }}" class="program-link">
-                  	{% endif %}
+                  {% if program.programpage.external_program_page_url %}
+                    <a href="{{ program.programpage.external_program_page_url }}" class="program-link">
+                  {% else %}
+                    <a href="{{ program.programpage.url }}" class="program-link">
+                  {% endif %}
                 {% endif %}
-                <div class="media-item">
-                  <div class="media-top">
-                    <div class="image-wrap">
+                    <div class="program-thumbnail-image-wrap">
                       {% if program.programpage and program.programpage.thumbnail_image %}
-                      {% image program.programpage.thumbnail_image fill-690x530 as thumbnail_image %}
-                      <img class="image" src="{{ thumbnail_image.url }}" alt="course image for {{program}}"
-                           width="{{ thumbnail_image.width }}" height="{{ thumbnail_image.height }}" />
+                        {% image program.programpage.thumbnail_image fill-345x265 as thumbnail_image %}
+                        <img src="{{ thumbnail_image.url }}" alt="course image for {{ program }}"
+                             width="{{ thumbnail_image.width }}" height="{{ thumbnail_image.height }}"/>
                       {% else %}
-                      <img class="image" src="{% static 'images/course-thumbnail.png' %}" alt="course image for {{program}}" width="378" height="225" />
-                      {% endif %}
-                    </div>
-                  </div>
-                  <div class="info-wrap">
-                    <div class="title">{{ program }}</div>
-                    <div class="description">
-                      {% if program.description %}
-                      {{ program.description }}
-                      {% else %}
-                      No description available for this program.
-                      {% endif %}
-                    </div>
-                    <div class="media-footer">
-                      {% if program.course_set.count == 1 %}
-                      <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} course</span></div>
-                      {% else %}
-                      <div class="widget-metas"><span class="meta-courses">{{ program.course_set.count }} courses</span></div>
+                        <img src="{% static 'images/course-thumbnail.png' %}"
+                             alt="course image for {{ program }}" width="345" height="265"/>
                       {% endif %}
+                      <div class="program-title">{{ program }}</div>
```

Comment:
It's not obvious from `ProgramPage` but each wagtail page includes a `title` already